docs/pages/project/contributing/meshery-server.md

```diff
@@ -104,3 +104,134 @@ github.com/layer5io/meshkit => ../meshkit
   </li>
 </ol>
 
+### Practicing Schema driven development for Meshery server.
+The guide covers instructions on how to generate Go code from OpenAPI specifications, best practices to ensure the generated code meets the requirements, follows the conventions of the project and necessary pre/post-requisites.
+
+### Installating required tools
+1. `oapi-codegen`: https://github.com/oapi-codegen/oapi-codegen?tab=readme-ov-file#install
+2. `redocly`: `npm i -g @redocly/cli`
+
+## Prerequisite
+1. Ensure you have forked the [meshery/schemas](github.com/meshery/schemas) and working directory is `schemas` repository.
+2. Resolving References in JSON Schemas:
```

Comment:
This step is out of place; not well-described. There's no lead-in or explanation.